main.go

```diff
@@ -124,17 +124,22 @@ func main() {
 
 	if load {
 		ctx.paClient = paClient
-		if sourceName == "" {
-			fmt.Fprintf(os.Stderr, "No source specified to load.\n")
-			os.Exit(1)
-		}
+		sources := getSources(paClient)
 
 		if supressorState(paClient) != unloaded {
 			fmt.Fprintf(os.Stderr, "Supressor is already loaded.\n")
 			os.Exit(1)
 		}
 
-		sources := getSources(paClient)
+		if sourceName == "" {
+			defaultSource, err := getDefaultSourceID(paClient)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "No source specified to load and failed to load default source: %+v\n", err)
+				os.Exit(1)
+			}
+			fmt.Printf("No source specified to load, loading default source with Device ID %s\n\n", defaultSource)
```

Comment:
I don't think we should print that here. If the user does that, they (probably) know what they want to do and I'd prefer noisetorch follow the philosophy of not printing anything on success. I think we should instead update the help text (it's in the flag parsing).

Reply:
I agree.
Updated the help text for the load flag.